### DISCUSSION TITLE
Please update image with no vulnerability

### DISCUSSION THREAD

**Discussion by subodh624** [supabase/supabase#30967] [Feature Requests]
I have been working with the following images, but my scans using Trivy reveal numerous high and critical vulnerabilities with latest tag version too, including in standard library packages:

```
supabase/postgres:15.1.0.147
supabase/studio:20240326-5e5586d
supabase/gotrue:v2.143.0
postgrest/postgrest:v12.0.1
supabase/realtime:v2.27.5
supabase/postgres-meta:v0.80.0
supabase/storage-api:v0.46.4
```
I've been working on rebuilding the following images to address high and critical vulnerabilities identified by Trivy scans. However, I am struggling to resolve vulnerabilities related to standard library (stdlib) packages.

Can the repository maintainers or community contributors provide updated versions of these images with no vulnerabilities? Alternatively, are there any recommendations for achieving a secure image rebuild, particularly addressing stdlib-related vulnerabilities?

**Comment by encoding-muzzu** [comment]
Yes for me also same, that's the reason we are switching from supabase to traditional one
